OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -267,10 +266,7 @@ public interface INotifyDeployTriggered
 	public interface IAcceptResourcesInfo : ITraitInfoInterface { }
 	public interface IAcceptResources
```

Comment:
Is this interface even still needed? Can we remove it? Should we remove it?

Reply:
`IAcceptResourcesInfo` is unneeded. But `IAcceptResources` is still used